app/main.py

```diff
@@ -13,7 +13,7 @@ def go_to_cafe(friends: list, cafe: Cafe):
         except NotWearingMaskError:
             masks_to_buy += 1
 
-    if not masks_to_buy:
+    if masks_to_buy > 0:
```

Comment:
Yes, you are wright, this should be "if masks_to_buy:" - without not. Also you don't need the last else statement. You can write just return f"Friends can go to {cafe.name}".
Good job!